README.md

```diff
@@ -1,20 +1,25 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+It binds an event handler to a click event - e.g, it connects to something that does something when the user clicks
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+It should be in your (document).ready function
+So it can access all functions/variables elsewhere
```

Comment:
You're correct that click handlers should be in your $(document).ready() but your reason isn't quite right. They need to be there because they depend on all the HTML elements (and their IDs) to be fully loaded - i.e. for the HTML document to be ready - before we can select them with JQuery and bind event handlers to them.